tool/data/config/logs/config.go

```diff
@@ -34,5 +35,8 @@ func (config *Config) ToMap(ctx *runtime.Context) (string, map[string]interface{
 	if config.Encoding != "" {
 		resultMap["encoding"] = config.Encoding
 	}
+	if config.Retention != 0 {
```

Comment:
Should this be <= 0 to account for negative values or is that checked beforehand?